data/formats-data.js

```diff
@@ -5489,7 +5496,7 @@ exports.BattleFormatsData = {
 		tier: "OU",
 	},
 	bunnelby: {
-		randomBattleMoves: ["agility", "earthquake", "return", "quickattack", "uturn", "stoneedge", "spikes", "bounce"],
+		randomBattleMoves: ["earthquake", "return", "quickattack", "uturn", "stoneedge", "spikes"],
```

Comment:
dont run spikes and stone edge, run wild charge and thief instead.

Reply:
spikes is mighty hard to deny in general but i can axe stone edge and add one or both of those

Reply:
bunnelby should just hit hard using huge power and that's it

Reply:
but spikes is soooooo good here